src/resolver.js

```diff
@@ -4,14 +4,17 @@ import {normalizeSwagger} from './helpers'
 
 export function makeFetchJSON(http) {
   return (docPath) => {
-    return http({
+    return Promise.resolve(http({
```

Comment:
Such that we can mock, without having to provide `.json`
eg
```js
http({ one: 1})
// vs
http({ json() { return Promise.resolve({ one: 1 }) })
```
Given that we're using `xmock` throughout... we don't strictly need mocking the `http` function anymore. I'll revise this...